lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -165,8 +165,7 @@ public static ParquetEncoding getParquetEncoding(Encoding encoding)
             case RLE:
                 return ParquetEncoding.RLE;
             case BYTE_STREAM_SPLIT:
-                // TODO: https://github.com/trinodb/trino/issues/8357
-                throw new ParquetDecodingException("Unsupported Parquet encoding: " + encoding);
+                return ParquetEncoding.BYTE_STREAM_SPLIT;
             case BIT_PACKED:
```

Comment:
Let's replace this switch case with enhanced switch case now that we're covering all encodings

Reply:
Done.